db/schema.rb

```diff
@@ -397,6 +397,18 @@
     t.index ["name"], name: "index_roles_on_name"
   end
 
+  create_table "tasks", force: :cascade do |t|
+    t.string "activity_message_value", limit: 65536, null: false
+    t.string "job_class", limit: 255, null: false
+    t.datetime "job_enqueued_for_retry_at"
```

Comment:
job_enqueued_for_retry_at won't really come into play until the TaskJobMonitor happens. I used it for thinking thru statuses tho, and have a placeholder for :stalled. Should I remove this and the :stalled code for now?

Reply:
No, it's fine, it'll come into play soon enough.